app/views/coronavirus_form/check_your_answers/sections/_space.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  title: t("coronavirus_form.check_your_answers.sections.space.title"),
+  heading_level: 2,
+  heading_size: "l",
+  items: question_items("offer_space"),
+  edit: {
+    href: "offer-space?change-answer",
+  },
+} %>
+
+<% unless session[:offer_space] == I18n.t("coronavirus_form.questions.offer_space.options.option_no.label") %>
+  <%= render "govuk_publishing_components/components/summary_list", {
+    title: t("coronavirus_form.check_your_answers.details"),
+    heading_level: 3,
+    heading_size: "m",
+    items: space_items,
+    edit: {
+      href: "offer-space?change-answer",
```

Comment:
This should be `offer-space-type?change-answer`

Reply:
Whoops!  Great spot - thanks @Rosa-Fox 👍